src/dataunion/Contracts.ts

```diff
@@ -53,11 +61,15 @@ export class Contracts {
     }
 
     getDataUnionSidechainAddress(mainnetAddress: EthereumAddress) {
-        if (!this.factorySidechainAddress) {
-            throw new Error('StreamrClient has no factorySidechainAddress configuration.')
+        if (!isAddress(this.factorySidechainAddress)) {
+            throw new Error('StreamrClient factorySidechainAddress configuration is ' + this.factorySidechainAddress ? 'missing' : 'not a valid Ethereum address')
```

Comment:
I think this ternary is around the wrong way, 

```suggestion
            throw new Error('StreamrClient factorySidechainAddress configuration is ' + this.factorySidechainAddress ? 'not a valid Ethereum address' : 'missing')
```